app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -569,3 +594,13 @@ open class BrowserActivity : DuckDuckGoActivity(), CoroutineScope by MainScope()
         val newInstanceState: Bundle?
     )
 }
+
+// Temporary class to keep track of latest visited tabs, keeping unique ids.
```

Comment:
"Temporary" makes me think this is temporary code that will be changed soon. Could we update the comment to indicate it's the data that is only held for a short while? (e.g., "used to keep track of the latest visited tabs in memory")

Reply:
Yes, the comment refers that is a class that will be changed soon, not related with the lifecycle of the data stored. The plan is: 1) validate assumption around transaction too large exception  2) put some more love into the code. If 1 fails, we will remove it. If it works, we can then extract this into a wrapper that handles fragmentManager + memory list.